README.md

```diff
@@ -46,7 +46,10 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [DrunkDevs](#drunkdevs) - **streaming:** Game Development, Game Maker: Studio, Aseprite
 - [Eddie Jaoude](#eddie-jaoude) - **streaming:** Open Source, JavaScript, Angular, AWS, Serverless, Node, TypeScript, GitHub, Cucumber
 - [Erik Guzman](#erik-guzman) - **streaming:** JavaScript, Node.js, ReactJS, React Native, APIs, Web Development
-- [Feross Aboukhadijeh](#feross-aboukhadijeh) - **streaming:** JavaScript, Node.js, Frameworks, Developer Tooling, Web Development
+- [Feross Aboukhadijeh](#feross-aboukhadijeh) - **streaming:** JavaScript, Node.js, Frameworks, Developer Tooling, Web 
+Game Development
+- [WindybeardGames](#windybeardgames) - **streaming:** Game Development (various), ClickTeam Fusion, PhotoShop, Spine
```

Comment:
The index of names is alphabetically sorted. Could you please insert WindybeardGames toward the end of the index where it belongs?